### PR TITLE
Add New Port Haunted Lands Burial Grounds

### PR DESCRIPTION
**Pull request by MadShmupper**
Add New Port Haunted Lands Burial Grounds

# New Port for { Haunted Lands Burial Grounds }

## Game Information
- **Title**: { Haunted Lands Burial Grounds
- **URL**: { https://alevgor.itch.io/haunted-lands-burial-grounds }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/hauntedlandsburial/Haunted Lands Burial Grounds.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+export GMLOADER_PLATFORM="os_linux"
+
+cd "$GAMEDIR"
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Check if .exe exists
+if [ -f "./gamedata/Haunted Lands Burial Grounds 1.1.0.exe" ]; then
+    # Extract its contents in place using 7zzs
+    ./libs/7zzs x "./gamedata/Haunted Lands Burial Grounds 1.1.0.exe" -o"./gamedata/"
```
Comment (cdeletre) [inline ports/hauntedlandsburial/Haunted Lands Burial Grounds.sh:33]: If something goes wrong with extraction I think it's better to stop here
```suggestion
    ./libs/7zzs x "./gamedata/Haunted Lands Burial Grounds 1.1.0.exe" -o"./gamedata/" || exit 1
```

File: ports/hauntedlandsburial/Haunted Lands Burial Grounds.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+fi
+
+# Rename data.win to game.droid if it exists
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+
+$GPTOKEYB "gmloader" -c "./hauntedlandsburial.gptk" &
+echo "Loading, please wait... " > /dev/tty0
+
+$ESUDO chmod +x "$GAMEDIR/gmloader"
+pm_platform_helper $GAMEDIR/gmloader
```
Comment (cdeletre) [inline ports/hauntedlandsburial/Haunted Lands Burial Grounds.sh:67]:
```suggestion
pm_platform_helper "$GAMEDIR/gmloader"
```

File: ports/hauntedlandsburial/hauntedlandsburial/hauntedlandsburial.zip
Comment (cdeletre) [inline ports/hauntedlandsburial/hauntedlandsburial/hauntedlandsburial.zip:1]: Rename to `.apk` and remove these files from the archive using `zip -d archive.apk file1 file2`:)
- AndroidManifest.xml
- META-INF/CERT.RSA
- META-INF/CERT.SF
- META-INF/MANIFEST.MF
- classes.dex
- resources.arsc

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/hauntedlandsburial/Haunted Lands Burial Grounds.sh (hunk 2 above)
Comment (MadShmupper) [reply ports/hauntedlandsburial/Haunted Lands Burial Grounds.sh]: Oopsies😂
I just sent change through. Hopefully fixed what ya requested. Thank you🎩🎮

File: ports/hauntedlandsburial/hauntedlandsburial/hauntedlandsburial.zip
Comment (MadShmupper) [reply ports/hauntedlandsburial/hauntedlandsburial/hauntedlandsburial.zip:1]: Lol.. Th. I hope it trimmed successfully.

**Review by cdeletre — APPROVED** [review]

File: ports/hauntedlandsburial/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## Notes
+
+Thanks to alevgor for creating this lovely game & thanks to all who tested and were involved in this port!
```
Comment (Cebion) [inline ports/hauntedlandsburial/README.md:3]: URL to game is missing.

**Review by Cebion — APPROVED** [review]

**Pull request merged**